### ISSUE TITLE
[TF2] [Feature Request] Strange variants should be added for all weapons

### ISSUE THREAD

**Issue by agrastiOs** [ValveSoftware/Source-1-Games#2083]
Strange variants should be added to weapons like Nessie's Nine Iron, Sharp Dresser, or Short Circuit. Community has asked for more for years.

**Comment by Haze33E** [comment]
Yeah they need to re-release the salvaged crates again too. I'd love to own a strange gunslinger but I'm not paying $50+ USD for one.
